metaci/db/management/commands/purge_old_results.py

```diff
@@ -13,22 +14,44 @@ class Command(BaseCommand):
     def handle(self, *args, **options):
         year_ago = timezone.now() - timedelta(days=365)
 
+        def show_progress(source, batch_size=1000):
+            i = 0
+            for item in source:
+                i += 1
+                yield item
+                if not i % batch_size:
```

Comment:
This just outputs until `i` exceeds batch size. Did we want to have anything occur when `len(source) > batch_size` ?

Reply:
It outputs every time i is evenly divisible by batch_size.